src/pull.go

```diff
@@ -22,6 +22,7 @@ import (
 	"path/filepath"
 	"runtime"
 	"sort"
```

Comment:
I implemented all that but I still think we should write the "Clashes were fixed, try pulling again." when we finish this task so that user actually know that his push/pull did not happen - we only renamed the files. Your solution does not provide that.